<ui version="4.0" >
 <class>dlgabout</class>
 <widget class="QWidget" name="dlgabout" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <property name="sizeConstraint" >
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="lbl_menu_top_2" >
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet" >
        <string notr="true" >border-radius: 0px;
border: none;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #90b8d7, stop: 1 #0c669d);
color: #ffffff;</string>
       </property>
       <property name="text" >
        <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;span style=" font-weight:600;">KBFX: People behind the KBFX project&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
       </property>
       <property name="margin" >
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Maximum" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">Here is the people behind the KBFX application.&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">We would like to thank all people who have helped with KBFX or have donated money to our project!&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">Thanks a lot all of you out there!!! =)&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;/p>&lt;/body>&lt;/html></string>
       </property>
       <property name="margin" >
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Maximum" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>The Team</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <item>
         <widget class="QTextEdit" name="textEdit" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="MinimumExpanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="readOnly" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2" >
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" >
            <item>
             <widget class="QToolButton" name="toolButton_1" >
              <property name="minimumSize" >
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="images.qrc" >
                <normaloff>:/kbfx_team/images/sirajpc.png</normaloff>:/kbfx_team/images/sirajpc.png</iconset>
              </property>
              <property name="iconSize" >
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="checkable" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_2" >
              <property name="minimumSize" >
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="images.qrc" >
                <normaloff>:/kbfx_team/images/phobos-mars-small.jpg</normaloff>:/kbfx_team/images/phobos-mars-small.jpg</iconset>
              </property>
              <property name="iconSize" >
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="checkable" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_3" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="images.qrc" >
                <normaloff>:/kbfx_team/images/nookiepc.png</normaloff>:/kbfx_team/images/nookiepc.png</iconset>
              </property>
              <property name="iconSize" >
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="checkable" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" >
            <item>
             <widget class="QToolButton" name="toolButton_4" >
              <property name="minimumSize" >
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="images.qrc" >
                <normaloff>:/kbfx_team/images/nathanael.png</normaloff>:/kbfx_team/images/nathanael.png</iconset>
              </property>
              <property name="iconSize" >
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="checkable" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_5" >
              <property name="minimumSize" >
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="images.qrc" >
                <normaloff>:/kbfx_team/images/scarebyte.jpg</normaloff>:/kbfx_team/images/scarebyte.jpg</iconset>
              </property>
              <property name="iconSize" >
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="checkable" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_6" >
              <property name="minimumSize" >
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="images.qrc" >
                <normaloff>:/kbfx_team/images/ephracis.png</normaloff>:/kbfx_team/images/ephracis.png</iconset>
              </property>
              <property name="iconSize" >
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="checkable" >
               <bool>true</bool>
              </property>
              <property name="checked" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" >
            <property name="topMargin" >
             <number>0</number>
            </property>
            <item>
             <widget class="QToolButton" name="toolButton_7" >
              <property name="minimumSize" >
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="images.qrc" >
                <normaloff>:/kbfx_team/images/finalpf3.jpg</normaloff>:/kbfx_team/images/finalpf3.jpg</iconset>
              </property>
              <property name="iconSize" >
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="checkable" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_8" >
              <property name="minimumSize" >
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="text" >
               <string/>
              </property>
              <property name="iconSize" >
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="checkable" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_9" >
              <property name="minimumSize" >
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="text" >
               <string/>
              </property>
              <property name="iconSize" >
               <size>
                <width>90</width>
                <height>90</height>
               </size>
              </property>
              <property name="checkable" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_2" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Maximum" hsizetype="MinimumExpanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">Tip!&lt;/span> Hoover over the contact images to see each&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">personal contact information&lt;/p>&lt;/body>&lt;/html></string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
            <property name="margin" >
             <number>5</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer" >
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Maximum" hsizetype="MinimumExpanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title" >
             <string>Sponsors!</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3" >
             <item>
              <widget class="QLabel" name="label_3" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Maximum" hsizetype="MinimumExpanding" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text" >
                <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">We would like to thank swedish computer company PC&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">Agenten for sponsoring us with a server and bandwidth.&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">http://www.pcagenten.se&lt;/p>&lt;/body>&lt;/html></string>
               </property>
               <property name="wordWrap" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc" />
 </resources>
 <connections/>
</ui>
